wire-ios-data-model/Source/Repositories/FeatureRepository.swift

```diff
@@ -307,7 +309,40 @@ public class FeatureRepository: FeatureRepositoryInterface {
         }
     }
 
-    // MARK: - MLS Migration
+    // MARK: - E2EId
+
+    public func fetchE2EI() -> Feature.E2EI {
+        guard
+            let feature = Feature.fetch(name: .e2ei, context: context),
+            let featureConfig = feature.config
+        else {
+            return .init()
+        }
+
+        let config = try! JSONDecoder().decode(Feature.E2EI.Config.self, from: featureConfig)
+        return .init(status: feature.status, config: config)
+    }
+
+    public func storeE2EI(_ e2ei: Feature.E2EI) {
+        let config = try! JSONEncoder().encode(e2ei.config)
```

Comment:
suggestion: avoid force try like we do in other methods in this class.

Reply:
https://github.com/wireapp/wire-ios/pull/1033